docs/issuer/README.md

```diff
@@ -8,3 +8,11 @@ Issuer application will:
 - retrieve user data from content management system
 - call [edge service](https://github.com/trustbloc/edge-service) to create student card verifiable credential on behalf of the issuer
 - present student card verifiable credential to the user
+- create the QR Code for retrieving the verifiable creadential 
+  
+   Note: In order to scan the QR code from your phone browser you need to be in the same network where your application is running. 
+     - Replace the 127.0.0.1 to an IP address of your network (for example wifi IP address) . 
```

Comment:
You have an extra space before the last period

Reply:
What do you think of
`Replace "127.0.0.1" in the following files to the IP address of the machine that your application is running on:`

instead?